### DISCUSSION TITLE
Is it possible to have one domain for two apiato apps (in different folders)?

### DISCUSSION THREAD

**Discussion by nozzy177** [apiato/apiato#753] [Q&A]
Is it possible to have one domain for two apiato apps (in different folders)?
I use Apache2 web-server .

Like this:
1. http://api.mysite.com/project1 -> /var/www/html/project1/apiato
2. http://api.mysite.com/project2 -> /var/www/html/project2/apiato

**Comment by arifrahmanfauzi** [comment]
hi @nozzy177 i think you need to little bit modify the nginx or apache web server .conf to direct incoming traffic using proxy pass i think
